src/actions/eventsActions.js

```diff
@@ -23,7 +23,7 @@ export const checkIn = (attendanceCode) => async dispatch => {
         'Content-Type': 'application/json',
         Authorization: `Bearer ${Storage.get('token')}`,
       },
-      body: JSON.stringify({ attendanceCode }),
+      body: JSON.stringify({ attendanceCode: info.attendanceCode, asStaff: info.asStaff }),
```

Comment:
maybe use a [`pick`](https://underscorejs.org/#pick) function here? e.g. `JSON.stringify(pick(info, ['attendanceCode', 'asStaff']))`

Reply:
I think this works fine, using pick means we would have to import another library when there's no serious need